### PR TITLE
HOMS-182 Add development installation instruction

### PR DESCRIPTION
**Pull request by lemonpaul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,58 @@ There are 2 ways to install homs.
...
+      type: 'static/activiti'
+  ```
+8. Add environment variable $HOMS with path to your homs folder:
+
+  ```
+  HOMS=/path/to/homs export HOMS
+  ```
+
+9. Run docker-compose:
+  ```
```
Comment (sugarfree1) [inline README.md]: New line before code block.

File: README.md [hunk 2]
```diff
@@ -55,6 +55,58 @@ There are 2 ways to install homs.
...
+    password: homs
+  ```
+7. Add in config/sources.yml
+
+  ```
+  sources:
+    bpmanagementsystem:
+      type: 'static/activiti'
+  ```
+8. Add environment variable $HOMS with path to your homs folder:
```
Comment (sugarfree1) [inline README.md]: $HOMS_PATH

File: docker-compose.yml [hunk 3]
```diff
@@ -49,12 +49,7 @@ services:
       - db_homs:postgres-homs
       - activiti_homs:activiti
     volumes:
-      - /etc/hydra/homs/homs_configuration.yml:/opt/homs/config/homs_configuration.yml
-      - /etc/hydra/homs/imprint.yml:/opt/homs/config/imprint.yml
-      - /etc/hydra/homs/hbw.yml:/opt/homs/config/hbw.yml
-      - /etc/hydra/homs/database.yml:/opt/homs/config/database.yml
-      - /etc/hydra/homs/activiti.yml:/opt/homs/config/activiti.yml
-      - /etc/hydra/homs/sources.yml:/opt/homs/config/sources.yml
+      - $HOMS:/opt/homs
```
Comment (sugarfree1) [inline docker-compose.yml]: Create new docker-compose.dev.yml.
Original docker-compose.yml should remain the same.

File: README.md (hunk 1 above)
Comment (lemonpaul) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (lemonpaul) [reply README.md]: Changed.

File: docker-compose.yml (hunk 3 above)
Comment (lemonpaul) [reply docker-compose.yml]: Created.

File: README.md [hunk 4]
```diff
@@ -55,6 +55,59 @@ There are 2 ways to install homs.
...
+8. Add environment variable $HOMS_PATH with path to your homs folder:
+
+  ```
+  HOMS_PATH=/path/to/homs export HOMS_PATH
+  ```
+
+9. Run docker-compose:
+
+  ```
+  docker-compose up -d
```
Comment (sugarfree1) [inline README.md]: Now, when you changed docker-compose.yml to docker-compose.dev.yml this step is wrong. You need to pass docker-compose.dev.yml as a parameter.

Comment (lemonpaul) [reply README.md]: Changed.

**Pull request merged**